diff --git a/common/console.cpp b/common/console.cpp
index 078a8d67..5c409f00 100644
--- a/common/console.cpp
+++ b/common/console.cpp
@@ -416,10 +416,12 @@ namespace console {
                 widths.push_back(width);
             }
 
-            if (!line.empty() && (line.back() == '\\' || line.back() == '/')) {
-                set_display(prompt);
-                replace_last(line.back());
-                is_special_char = true;
+            if (!line.empty() && (line.back() == '/')) {
+            	if (line.length() == 1) {
+	                set_display(prompt);
+	                replace_last(line.back());
+	                is_special_char = true;
+                }
             }
         }
 
@@ -430,18 +432,12 @@ namespace console {
 
             char last = line.back();
             line.pop_back();
-            if (last == '\\') {
-                line += '\n';
-                fputc('\n', out);
-                has_more = !has_more;
-            } else {
                 // llama will just eat the single space, it won't act as a space
-                if (line.length() == 1 && line.back() == ' ') {
-                    line.clear();
-                    pop_cursor();
-                }
-                has_more = false;
+            if (line.length() == 1 && line.back() == ' ') {
+                 line.clear();
+                 pop_cursor();
             }
+            has_more = false;
         } else {
             if (end_of_stream) {
                 has_more = false;
diff --git a/ggml/src/ggml-vulkan/vulkan-shaders/get_rows_quant.comp b/ggml/src/ggml-vulkan/vulkan-shaders/get_rows_quant.comp
index cfd645a3..b9ce9012 100644
--- a/ggml/src/ggml-vulkan/vulkan-shaders/get_rows_quant.comp
+++ b/ggml/src/ggml-vulkan/vulkan-shaders/get_rows_quant.comp
@@ -32,7 +32,7 @@ void main() {
     const uint iybs = i00 - i00%QUANT_K; // dst block start index
     const uint y_offset = QUANT_R == 1 ? 1 : QUANT_K/2;
 
-    vec2 v = dequantize(ib, iqs, 0);
+    vec2 v = vec2(0,0);
     const vec2 dm = get_dm(ib, 0);
     v = v * dm.x + dm.y;
 
